src/lib/utils.spec.js

```diff
@@ -137,3 +138,27 @@ describe('getFolderLink', () => {
     })
   })
 })
+
+describe('relativeAge', () => {
+  // constants
+  const sec = 1000
+  const min = 60 * sec
+  it('should have the recorded response for age=3s', () => {
```

Comment:
Nit: "should return the right data for age=3s" is a bit easier to understand (I couldn't understand why "recorded" was used, I looked for a memoize in the implementation)